recipes/glib/all/test_package/CMakeLists.txt

```diff
@@ -6,4 +6,4 @@ conan_basic_setup()
 find_package(glib REQUIRED)
 
 add_executable(${PROJECT_NAME} test_package.cpp)
-target_link_libraries(${PROJECT_NAME} glib::glib-2.0)
+target_link_libraries(${PROJECT_NAME} ${CONAN_LIBS})
```

Comment:
This looks like a step back. Why changing it?

Reply:
Because it linked to a single component. I'd want to make sure all components link successfully, but I think this change is not sufficient, because the linker throws away libraries which are not referenced, whithout checking if all the symbol they are referencing are abailable or not.

Reply:
@Croydon Do you find it better like this ?